app/src/main/java/org/p2p/wallet/history/repository/HistoryTransactionConverter.kt

```diff
@@ -67,19 +66,23 @@ object TransactionConverter {
             destinationIconUrl = destinationData.iconUrl.orEmpty()
         )
 
+    private fun String.toBigDecimalFromLamports(decimals: Int): BigDecimal =
```

Comment:
Did you mean `toBigIntegerFromLamports`?

Reply:
return type is BigDecimal, so i think it's alright here

Reply:
Oh, yeah, my fault